TOMM20.py

```diff
@@ -0,0 +1,263 @@
+from Bio import SeqIO
+from Bio.SeqUtils import gc_fraction
+import os
+
+ALPHABET_FOR_DNA = {'A', 'T', 'G', 'C', 'a', 't', 'g', 'c'}
+ALPHABET_FOR_RNA = {'A', 'U', 'G', 'C', 'a', 'u', 'g', 'c'}
+ALPHABET_FOR_PROTEIN = set('FLIMVSPTAYHQNKDECWRG')
```

Comment:
Если занудствовать, то можно было унифицировать и сделать всё через set. 
Но это конечно "Откройте окно! Слишком душно"

```suggestion
ALPHABET_FOR_DNA = set('ATGCatgc')
ALPHABET_FOR_RNA = set('AUGCaugc')
ALPHABET_FOR_PROTEIN = set('FLIMVSPTAYHQNKDECWRG')
```